src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs

```diff
@@ -287,6 +288,31 @@ public static CorInfoTypeWithMod getParameterType(IntPtr thisHandle, CORINFO_CLA
             return (CorInfoTypeWithMod)corInfoType;
         }
 
+        [UnmanagedCallersOnly]
+        public static uint getObjectLayout(IntPtr thisHandle, CORINFO_CLASS_STRUCT_* inputType, void** layoutPtrPtr)
+        {
+            var _this = GetThis(thisHandle);
+
+            TypeDesc type = _this.HandleToObject(inputType);
+
+            List<FieldStoreLayout> layout = new();
+
+            _this._compilation.GetObjectLayoutInstructions(type, layout);
```

Comment:
Nit: cleaner to return the list from `GetObjectLayoutInstructions` instead of mutating it?